### ISSUE TITLE
Hall-of-Fame/ Achievements Page for techspace members

### ISSUE THREAD

**Issue by silent-lad** [techspaceusict/techspaceusict.github.io#7]
we want to add a new achievements page on this website. Work on the design, MAKE A PR.

**Comment by khush31** [comment]
@silent-lad can i work on this ?

**Comment by silent-lad** [comment]
@khush31 yes. Just one thing make sure it compliments the color scheme of the current website.

**Comment by khush31** [comment]
> @khush31 yes. Just one thing make sure it compliments the color scheme of the current website. 

Compliment the colour scheme means change the colour theme right ??
